hybrasyl/World.cs

```diff
@@ -2376,11 +2474,371 @@ private void PacketHandler_0x30_MoveUIElement(Object obj, ClientPacket packet)
         private void PacketHandler_0x3B_AccessMessages(Object obj, ClientPacket packet)

```

Comment:
realistically, the switches should become individual overloaded methods which react based on the case. this will help with debugging code.